server/seed.js

```diff
@@ -0,0 +1,39 @@
+require("dotenv").config();
+const connectDB = require("./config/dbConnection");
+const bcrypt = require("bcrypt");
+const {
+  AdminUserDetails,
+} = require("./models/adminUserDetails/adminUserDetails");
+
+//Connect Mongodb
+connectDB();
+
+async function createAdminUser() {
+  try {
+    // Admin user credentials
+    const adminUseremail = process.argv[2];
+    const adminPassword = process.argv[3];
+
+    if (!adminUseremail || !adminPassword) {
+      console.error("Usage: npm run seed:db <admin_username> <admin_password>");
+      process.exit(1);
+    }
+
```

Comment:
Can we add a condition to check if the email is already present or not ? If present throw an error.

Reply:
@iamurmani I have implemented that finding already existing mailid when admin creations.